my_enumerables methods created

Varun2323 opened the pull request:
Enumerable:
   The  project aims at reimplementing  some basic ruby enumarables which are listed below:
- my_each?
- my_each_with_index?
- my_all?
- my_none?
- my_any?
- my_select
- my_count
- my_map
- my_inject

Varun2323 committed:
Enumerables methods created

Varun2323 committed:
changes added

SpaYco reviewed (approved):
# Project Approved ✔️ 

great work on making all methods working as expected in one go, your project is now approved!

things to note: 

- [x] in your next projects, please include more commits, for example, in this project, you could've had one commit for each method you add.

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

carloso01142 committed:
add my_each my_each_with_index and my_select tests

Varun2323 committed:
rspec created for Enumerables

here we have tested and created the rspec methods to check the enumerables results are valid for further functionality or not.

carloso01142 committed:
add multiply_els  test

NtwaliHeritier reviewed (commented):
**Hello Team**,
The project is already approved, so there was no need to ask for another review. Just merge your branch and work on other projects.
Best of luck.

carloso01142 committed:
clean the branch
